fc/protocol.py

```diff
@@ -638,6 +638,7 @@ def set_model(self, model):
             # Create weblab model at path
             create_weblab_model(
                 path,
+                self.output_folder or temp_dir,
```

Comment:
Lots of tests are failing because the method expects a string but `self.output_folder` is an `OutputFolder` instance.